### ISSUE TITLE
Clearing specific quill fields using a button

### ISSUE THREAD

**Issue by phildeman** [quilljs/quill#2548]
I am fairly new to Quilljs.  So, please bare with me as I attempt to describe my needs.  I am using Quilljs 1.3.6.  I want to add a button on an HTML form that will clear specific quilljs fields.

If I use jquery on any non-quilljs field, all I have to do is create a button that calls a javascript function, that deletes the field(s).  For example:

<b>HTML</b>
&lt;label for="strengths"&gt;How strong are you?&lt;label&gt;
&lt;input type='text' name='strengths' id='strengths'&gt;

&lt;input type="button" onclick="clearTextField()"&gt;Clear Field&lt;/button&gt;

<b>Javascript File</b>
function clearTextField() {
   $('#strengths').val('');
}

How do I clear a quilljs field?  I would like to do something like this:

<b>HTML</b>
&lt;label for="strengths"&gt;How strong are you?&lt;label&gt;
&lt;input type="hidden" name="strengths"&gt;
&lt;div id="strengths-container"&gt;
      &lt;p&gt;Blah Blah Blah&lt;/p&gt;
&lt;/div&gt;
&lt;input type="button" onclick="clearTextField()"&gt;Clear Field&lt;/button&gt;

<b>Javascript File</b>
function clearTextField() {
   strengths.setContents([]);
}

However, this won't work, because:
var strengths = new Quill("#strengths-container", {
          modules: {
              toolbar: toolbaroptions
          },
          theme: "snow"
      });

function clearTextField() {
   $('#strengths').val('');
}
was created in a different function.  So, the scope of the object is limited to the that function.

Is there a way clear the contents of a field with quilljs, separate from the function that instantiated the object?

**Comment by benbro** [comment]
What do you mean by a field? Quill have full control of the DOM and expose an API to update the [contents](https://quilljs.com/docs/api/#content) and [formats](https://quilljs.com/docs/api/#formatting), you shouldn't access it directly.

**Comment by phildeman** [comment]
A form field is a container that users enter information into it.  The information can be saved or used as  part of a condition to navigate a form to perform different task.

In the example I provided, quilljs was used to create a 'field' called strengths-container, so that users can enter data into it.

I am aware that quilljs has APIs.  My question was, is there a way to clear the contents of a field created by quilljs.  Your response did not provide an example of how to use the Quill API outside of the javascript function that instantiated the field's object name.

Yes, I already tried strengths.deleteText(0);  and
strengths.deleteText(0,250);  Neigther API worked.

Sending me that document that provides minimal example did not help me resolve my issue.
